content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.tracking()'
+Description: 'Modifies the spacing of the characters in a Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.tracking()`** modifier method modifies the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
+
+## Syntax
+
+```pseudo
+Text("Some text")
```

Comment:
Yes, I actually much prefer this way of showing the syntax for the ViewModifiers. I'm going to create an issue to amend these for the other existing entries that don't already following this structure if you would be down to take that on. But no pressure, if not, I can go ahead and make that edit myself but do lmk!

Reply:
Sure, I can do that!